.github/workflows/CI.yml

```diff
@@ -929,6 +933,47 @@ jobs:
           cd testing-framework/terraform/containerinsight_eks_prometheus && terraform destroy -auto-approve
 
 
+  e2etest-containerinsight-eks:
+    runs-on: ubuntu-latest
+    needs: [ get-testing-suites, e2etest-release, e2etest-preparation ]
+    strategy:
+      max-parallel: 5
+      matrix: ${{ fromJson(needs.get-testing-suites.outputs.containerinsight-eks-matrix) }}
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.INTEG_TEST_AWS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.INTEG_TEST_AWS_KEY_SECRET }}
+          aws-region: us-west-2
+
+      - name: Set up JDK 1.11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.11
+
+      - name: Set up terraform
+        uses: hashicorp/setup-terraform@v1
+
+      - name: Check out testing framework
+        uses: actions/checkout@v2
+        with:
+          repository: 'aws-observability/aws-otel-collector-test-framework'
+          path: testing-framework
+
+      - name: Run testing suite on eks
+        run: |
+          if [[ -f testing-framework/terraform/testcases/${{ matrix.testcase }}/parameters.tfvars ]] ; then opts="-var-file=../testcases/${{ matrix.testcase }}/parameters.tfvars" ; else opts="" ; fi
+          cd testing-framework/terraform/containerinsight_eks && terraform init && terraform apply -auto-approve -lock=false $opts -var="aoc_version=${{ needs.e2etest-preparation.outputs.version }}" -var="testcase=../testcases/${{ matrix.testcase }}"
+
+      - name: Destroy resources
+        if: ${{ always() }}
+        run: |
+          cd testing-framework/terraform/containerinsight_eks && terraform destroy -auto-approve
```

Comment:
Could you take a look at https://github.com/aws-observability/aws-otel-collector/pull/443 to see if this line here would cause similar issue?

Reply:
I am merging the two test cases. So the fix should also work for container insight.